bundles/org.pitest.pitest-junit5-plugin/META-INF/MANIFEST.MF

```diff
@@ -2,7 +2,7 @@ Manifest-Version: 1.0
 Bundle-ManifestVersion: 2
 Bundle-Name: pitest-junit5-plugin
 Bundle-SymbolicName: org.pitest.pitest-junit5-plugin
-Bundle-Version: 0.14.0.qualifier
+Bundle-Version: 1.2.1
```

Comment:
This should keep ".qualifier" like we do for "org.pitest" to avoid releasing different binary bundles with the same version (we had this problem in the past); see the "org.pitest" project.